src/common/platform/win32/i_input.cpp

```diff
@@ -123,12 +123,16 @@ int BlockMouseMove;
 
 static bool EventHandlerResultForNativeMouse;
 
-
 EXTERN_CVAR(Bool, i_pauseinbackground);
 
-
 CVAR (Bool, k_allowfullscreentoggle, true, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 
+FARG(noidle, "Configuration", "Keep ZDoom running at high priority, even in background.", "",
+	"When the user presses Alt-Tab to switch from ZDoom to another window, ZDoom will normally"
+	" reduce its priority class so that it does not slow the rest of the system down. By"
+	" specifying this parameter, ZDoom will try to run at full speed even when it is not the"
+	" active window.");
+
```

Comment:
-noidle no longer has any effect, the priority changing code was removed in favor of restricting the game to 35 FPS in the background (which can, obviously, be overridden with a user cvar). Just a relic of ZDoom's NT days when games often fucked around with system settings and the like (though admittedly that isn't the case here it still did not allow the user to set the game's priority themselves at launch, it did not respect the user's preferences at the OS level, so it was removed).